docs/testing.md

```diff
@@ -121,40 +121,21 @@ Sauce Labs can be used to run tests remotely in the cloud.
 
 1. Log into [http://saucelabs.com/account](http://saucelabs.com/account).
 
-2. [Download Sauce Connect](https://wiki.saucelabs.com/display/DOCS/Basic+Sauce+Connect+Proxy+Setup#BasicSauceConnectProxySetup-SettingUpSauceConnect)
-
-3. Open a new Terminal window or tab and navigate to the downloaded SauceConnect folder.
-    If you place the folder in your Application's folder this might look like:
-
-    ```
-    cd /Users/<YOUR MAC OSX USERNAME>/Applications/SauceConnect
-    ```
-
-4. Copy step 3 from the the SauceLabs
-   [Basic Setup instructions](https://wiki.saucelabs.com/display/DOCS/Basic+Sauce+Connect+Proxy+Setup#BasicSauceConnectProxySetup-SettingUpSauceConnect)
-   and run that in your Terminal window.
-   Once you see `Sauce Connect is up` in the Terminal,
-   that means the tunnel has successfully been established
-
-    > The Terminal command should already have your Sauce username and access key filled in.
-      If it doesn't, make sure you're logged in.
-
-5. Update and uncomment the `SAUCE_USERNAME`, `SAUCE_ACCESS_KEY`,
+2. Update and uncomment the `SAUCE_USERNAME`, `SAUCE_ACCESS_KEY`,
    and `SAUCE_SELENIUM_URL` values in your `.env` file.
    The access key can be found on the Sauce Labs
    [user settings page](https://saucelabs.com/beta/user-settings).
 
-6. Reload the settings with `cd .. && cd cfgov-refresh`. Type `y` if prompted.
+3. Reload the settings with `cd .. && cd cfgov-refresh`. Type `y` if prompted.
```

Comment:
We could update this to `source .env`, which won't prompt the user I don't think and would also accommodate if they named their project folder differently.